djangocms_versioning/test_utils/blogpost/cms_config.py

```diff
@@ -1,8 +1,19 @@
 from cms.app_base import CMSAppConfig
 
-from .models import BlogPostVersion, CommentVersion
+from djangocms_versioning import Versionable
+
+from .models import BlogPost, BlogContent, Comment
 
 
 class BlogpostCMSConfig(CMSAppConfig):
     djangocms_versioning_enabled = True
-    versioning_models = [BlogPostVersion, CommentVersion]
+    versioning = [
+        Versionable(
+            grouper=BlogPost,
+            content=BlogContent,
+        ),
+        Versionable(
+            grouper=BlogPost,
+            content=Comment,
+        )
```

Comment:
I removed versioning definition for comment, because I'm not sure it's correct - having a BlogPost as a grouper for Comment doesn't make Comment model versionable - it doesn't allow for a single comment under a post to have multiple versions.